src/app/organization/organization-routing.module.ts

```diff
@@ -302,11 +304,24 @@ const routes: Routes = [
         },
         {
           path: 'entity-data-table-checks',
-          component: EntityDataTableChecksComponent,
           data: { title: extract('Entity Data Table Checks'), breadcrumb: 'Entity Data Table Checks' },
-          resolve: {
-            entityDataTableChecks: EntityDataTableChecksResolver
-          }
+          children: [
+            {
+              path: '',
+              component: EntityDataTableChecksComponent,
+              resolve: {
+                entityDataTableChecks: EntityDataTableChecksResolver
+              },
+            },
+            {
+              path: 'create',
+              component: CreateEnityDataTableChecksComponent,
+              data: { title: extract('Create Entity Data Table Checks'), breadcrumb: 'Create Entity Data Table Checks' },
```

Comment:
Bread Crumb too long
```suggestion
              data: { title: extract('Create Entity Data Table Checks'), breadcrumb: 'Create' },
```